must-gather/node-gather/daemonset.yaml

```diff
@@ -39,8 +39,10 @@ spec:
         volumeMounts:
           - name: sys
             mountPath: /host/sys
+            readOnly: true
           - name: proc
             mountPath: /host/proc
+            readOnly: true
           - name: podres
             mountPath: /host/podresources
```

Comment:
Do we write to podres?

Reply:
yes: we need to write the request messages to send them to the server.